packages/mui-material/src/InputBase/InputBase.js

```diff
@@ -184,12 +184,11 @@ export const InputBaseInput = styled('input', {
     display: 'block',
     // Make the flex item shrink with Firefox
     minWidth: 0,
-    width: '100%', // Fix IE11 width issue
+    flexGrow: 1,
     animationName: 'mui-auto-fill-cancel',
     animationDuration: '10ms',
     '&::-webkit-input-placeholder': placeholder,
     '&::-moz-placeholder': placeholder, // Firefox 19+
-    '&:-ms-input-placeholder': placeholder, // IE11
```

Comment:
This and the other `'&:-ms-input-placeholder'` were added in the original InputBase: https://github.com/mui/material-ui/pull/12076/files#diff-a5f495d99c59823944e8c99682e25d509e79d427ab0442c71589da33e6f350f6R81

Tested it in Chrome, Firefox, and Safari. Screenshot in Chrome:

https://github.com/mui/material-ui/assets/16889233/12cb9fc6-2f61-4739-9b4f-5cbaa0fdde7b